source/components/JsxParser.test.js

```diff
@@ -172,10 +184,9 @@ describe('JsxParser Component', () => {
     expect(div.nodeName).toEqual('DIV')
 
     const divChildren = Array.from(div.childNodes)
-    expect(divChildren).toHaveLength(3)
-    expect(divChildren.map(n => n.nodeType))
-      .toEqual([8, 3, 8])
-    expect(divChildren[1].textContent).toEqual('Non-Custom')
+    expect(divChildren).toHaveLength(1)
+    expect(divChildren[0].nodeType).toEqual(3)
+    expect(divChildren[0].textContent).toEqual('Non-Custom')
```

Comment:
I especially love that your changes fix this bug! Previously, the inner-text would wind up wrapped with comments - but this change resolves it far more elegantly.